frontend/lib/screens/new_student/local_widget/student_form.dart

```diff
@@ -0,0 +1,210 @@
+                        _createpasswordField(student),
+                        _createDescriptionField(student),
+                      ],
+                    ),
+                  ),
+                ),
+                Padding(
+                  padding: const EdgeInsets.symmetric(vertical: 16.0),
+                  child: RaisedButton(
+                    color: Colors.grey[600],
+                    onPressed: () {
+                      // Validate will return true if the form is valid, or false if
+                      // the form is invalid.
+                      if (_formKey.currentState.validate()) {
+                        widget.changeStep(false);
+                        
+                      }
+                    },
+                    child: Text(
+                      'Continue',
+                      style: TextStyle(color: Colors.white),
+                    ),
+                  ),
+                ),
+              ],
+            ),
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _createnameField(Student student) {
+    return TextFormField(
+      decoration: const InputDecoration(
+        hintText: 'Student name',
+      ),
+      initialValue: student.name ?? '',
+      onChanged: (value) {
+        setState(() {
+          student.name = value;
+        });
+      },
+      validator: (value) {
+        if (value.isEmpty) {
+          return 'Please enter a student name';
+        }
+        return null;
+      },
+    );
+  }
+
+Widget _createsurnameField(Student student) {
+    return TextFormField(
+      decoration: const InputDecoration(
+        hintText: 'Student surname',
+      ),
+      initialValue: student.surname ?? '',
+      onChanged: (value) {
+        setState(() {
+          student.surname = value;
+        });
+      },
+      validator: (value) {
+        if (value.isEmpty) {
+          return 'Please enter a student surname';
+        }
+        return null;
+      },
+    );
+  }
+
+  Widget _createemailField(Student student) {
+    return TextFormField(
+      decoration: const InputDecoration(
+        hintText: 'student@school.com',
+      ),
+      initialValue: student.email ?? '',
+      onChanged: (value) {
+        setState(() {
+          student.email = value;
+        });
+      },
+      validator: (value) {
+        if (value.isEmpty) {
+          return 'Please enter a student email';
+        }
+        return null;
+      },
+    );
+  }
+
+Widget _createpasswordField(Student student) {
+    return TextFormField(
+      decoration: const InputDecoration(
+        hintText: 'Student password',
+      ),
+      initialValue: student.password ?? '',
+      onChanged: (value) {
+        setState(() {
+          student.password = value;
+        });
+      },
+      validator: (value) {
+        if (value.isEmpty) {
+          return 'Please enter a student email';
+        }
+        return null;
+      },
+    );
+  }
```

Comment:
Why is this here? It should not be implemented in this screen

Reply:
Done